FormValidation.html

```diff
@@ -16,43 +16,41 @@
     </style>
     <script>
       function ValidateForm() {
-        name = document.forms["myForm"]["txtname"].value;
+        var name = document.forms["myForm"]["txtname"].value;
+        var mobile = document.forms["myForm"]["txtnumber"].value;
+        var mailformat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+        var mail = document.forms["myForm"]["txtemail"].value;
+        var place = document.forms["myForm"]["txtplace"].value;
+        var genders = document.getElementsByName("gender");
+        let x = 0;
+
         if (name == "") {
           alert("Enter the name");
           return false;
         }
-        var mobile = document.forms["myForm"]["txtnumber"].value;
         if (mobile == "") {
```

Comment:
please add validation to mobile number 
it must be 10 digit and start from [6,7,8,9]

Reply:
Validation is added to a mobile number and it contains 10 digits with  start from [6,7,8,9].